i18n/en.i18n.json

```diff
@@ -185,6 +185,8 @@
   "LDAP_Dn" : "Distinguished Name (DN)",
   "LDAP_Enable" : "Enable LDAP",
   "LDAP_Port" : "LDAP Port",
+  "LDAP_Sync_User_Data" : "Keep user data in sync with server",
+  "LDAP_Sync_User_Data_FieldMap" : "User data field map",
```

Comment:
I would love to if I have some time. I have experience with angular+coffeecript+ldap so I should be able to help out.

How should I get started with _Meteor_ and developing _RocketChat_?